internal/grpc_service_config/config_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package grpc_service_config
+
+import (
+	"testing"
+
+	duration "github.com/golang/protobuf/ptypes/duration"
+	wrappers "github.com/golang/protobuf/ptypes/wrappers"
+	"github.com/google/go-cmp/cmp"
+	code "google.golang.org/genproto/googleapis/rpc/code"
+	"google.golang.org/protobuf/proto"
+)
+
+func Test_Parse(t *testing.T) {
```

Comment:
```suggestion
func TestParse(t *testing.T) {
```

Underscores make sense to split off integration tests or something. But, normal tests are named like normal functions.

Reply:
Ok, thanks! Done.